README.md

```diff
@@ -19,7 +19,7 @@ To use the `mommy` function, you can source the `shell-mommy.sh` script in your
 . /path/to/shell-mommy.sh
 ```
 
-If you'd like it to always show a message after each command, you can define a custom `PROMPT_COMMAND` like so:
+If you'd like it to always show a message after each command, you can define a custom `PROMPT_COMMAND` like so (bash only):
```

Comment:
From a comment on the related issue, it's potentially easier than I thought to use `pre_cmd` to make it work. I'll try it and add it to the readme too.